test/integration/match_coverage_test.go

```diff
@@ -18,7 +18,7 @@ import (
 
 func getPackagesByPath(t *testing.T, theScope scope.Scope, catalog *pkg.Catalog, thePath string) []*pkg.Package {
 	t.Helper()
-	refs, err := theScope.FilesByGlob(thePath)
+	refs, err := theScope.Resolver.FilesByGlob(thePath)
```

Comment:
this was tricky to figure out... it was breaking because of changes in: https://github.com/anchore/syft/pull/148/